pkgs/build-support/rust/hooks/cargo-setup-hook.sh

```diff
@@ -1,6 +1,8 @@
 cargoSetupPostUnpackHook() {
     echo "Executing cargoSetupPostUnpackHook"
 
+    export NIX_LDFLAGS+=" @aarch64LinuxGccWorkaround@"
```

Comment:
A fixme should be added that explains when this can be removed, I think. In general a bit of background in a comment is a good idea, IMO.

Reply:
Background is provided both in the commit message + where this substitution is defined. I'm not sure if adding a duplicate comment is worthwhile?